website/docs/security/secrets-management/secrets-manager/ascp.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "AWS Secrets and Configuration Provider (ASCP)"
+sidebar_position: 62
+---
+
+If you ran the `prepare-environment` script detailed in a [previous step](./index.md), it has already installed the AWS Secrets and Configuration Provider (ASCP) for the Kubernetes Secrets Store CSI Driver that's required for this lab.
+
+Lets then, validate if the addons deployed.
+
+Check the Secret Store CSI drive `DaemonSet` and respective `Pods`.
+
+```bash
+$ kubectl -n secrets-store-csi-driver get pods,daemonsets -l app=secrets-store-csi-driver
+NAME                                                        DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR            AGE
+daemonset.apps/csi-secrets-store-secrets-store-csi-driver   3         3         3       3            3           kubernetes.io/os=linux   3m57s
+
+NAME                                                   READY   STATUS    RESTARTS   AGE
+pod/csi-secrets-store-secrets-store-csi-driver-bzddm   3/3     Running   0          3m57s
+pod/csi-secrets-store-secrets-store-csi-driver-k7m6c   3/3     Running   0          3m57s
+pod/csi-secrets-store-secrets-store-csi-driver-x2rs4   3/3     Running   0          3m57s
+```
+
+Check the CSI Secrets Store Provider for AWS driver `DaemonSet` and respective `Pods`.
+
+```bash
+$ kubectl -n kube-system get pods,daemonset -l "app=secrets-store-csi-driver-provider-aws"  
+NAME                                                   DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR            AGE
+daemonset.apps/secrets-store-csi-driver-provider-aws   3         3         3       3            3           kubernetes.io/os=linux   2m3s
+
+NAME                                              READY   STATUS    RESTARTS   AGE
+pod/secrets-store-csi-driver-provider-aws-4jf8f   1/1     Running   0          2m2s
+pod/secrets-store-csi-driver-provider-aws-djtf5   1/1     Running   0          2m2s
+pod/secrets-store-csi-driver-provider-aws-dzg9r   1/1     Running   0          2m2s
+```
+
+You should also see that we already have created a SecretProviderClass, which is a namespaced custom resource that's used provide driver configurations and specific parameters to access your secrets stored in AWS Secrets Manger via CSI driver.
```

Comment:
I'd suggest we get the user to create this, makes it easier for them to browse the YAML to see how it happened.